.github/workflows/build-images.yml

```
@@ -0,0 +1,240 @@
+
+  # One job looping over the images, not a matrix: each leg was a whole runner
+  # VM doing a login plus one imagetools call, so boot/checkout dwarfed the real
+  # work and burned ~19 org runner slots every nightly. Per-image isolation is
+  # kept in the loop — one image failing skips only that image.
+  manifest:
+    name: Stitch multi-arch tags
+    needs: [enumerate, build]
+    if: ${{ !cancelled() }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker/login-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)